q2_types_genomics/reference_db/_type.py

```diff
@@ -14,4 +14,9 @@
 Diamond = SemanticType('Diamond', variant_of=ReferenceDB.field['type'])
 Eggnog = SemanticType('Eggnog', variant_of=ReferenceDB.field['type'])
 
-plugin.register_semantic_types(ReferenceDB, Diamond, Eggnog)
+EggnogSequenceTaxa = SemanticType(
```

Comment:
Sorry @Sann5, I just noticed this now... Wouldn't it make sense to change this name too to `EggnogProteinSequences` (similarly to how you did it on the format)? I was just looking at the corresponding action in q2-moshpit and must say it really reads strange when you see `ReferenceDB[EggnongSequenceTaxa]`...